src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -1,20 +1,17 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Collections;
+using System.ComponentModel;
```

Comment:
why is new namespace needed?

Reply:
[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)] ->         [EditorBrowsable(EditorBrowsableState.Never)]